models/articles.model.js

```diff
@@ -0,0 +1,64 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+exports.selectAllArticles = async (
+  topic = "%%",
+  sort_by = "created_at",
```

Comment:
Please include some way of checking whether the sort_by or topic passed in is valid/accepted, and if not, return an error!